docs/en/ingest-management/release-notes/release-notes-8.8.asciidoc

```diff
@@ -22,12 +23,46 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.8.2 relnotes
+
+[[release-notes-8.8.2]]
+== {fleet} and {agent} 8.8.2
+
+Review important information about the {fleet} and {agent} 8.8.2 release.
+
+[discrete]
+[[security-updates-8.8.2]]
+=== Security updates
+
+{agent}::
+* Updated Go version to 1.19.10. {agent-pull}2846[#2846] 
+
+[discrete]
+[[enhancements-8.8.2]]
+=== Enhancements
+
+* Log start and stop operations from service runtime at `INFO` rather than `DEBUG` level. {agent-pull}2879[#2879] {agent-issue}2864[#2864]
+
+[discrete]
+[[bug-fixes-8.8.2]]
+=== Bug fixes
```

Comment:
Looks like this entry from https://github.com/elastic/elastic-agent/pull/2939/files got dropped:

> * Fix logs collection In diagnostics when {agent} is running on K8s. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2905[#https://github.com/elastic/elastic-agent/pull/2905] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/2899[#https://github.com/elastic/elastic-agent/issues/2899]